lib/jsdom/virtual-console.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+var EventEmitter = require("events").EventEmitter;
+var utils = require("./utils");
+
+function VirtualConsole () {
```

Comment:
No need for space after the function name. My rule is `function<space><name>()`, where if `<name>` is the empty string, that becomes `function<space>()`